packages/sdk/src/index.ts

```diff
@@ -133,15 +133,20 @@ export default class SDK {
       "success"
     );
     this.requestCache.removeRequest(matchingRequest.request);
```

Comment:
can we add a "response time" log here? by comparing the time the request was created with the time the response was found

Reply:
Good idea! Next commit has the addition.